lib/processor.js

```diff
@@ -497,6 +497,9 @@ class Processor {
         return true
       }
     }
+    if (node.type === 'decl' && node.prop === 'content') {
```

Comment:
```suggestion
    if (node.prop === 'content') {
```

Only `Declaration` has `prop`.